src/components/GroupBanner/GroupBanner.js

```diff
@@ -74,8 +74,9 @@ export default function GroupBanner ({
 
         {group && <div styleName='right'>
           <Link
-            styleName={cx({ 'about': true, 'is-about-open': isAboutOpen })}
+            styleName={cx({ about: true, 'is-about-open': isAboutOpen })}
             to={isAboutOpen ? groupUrl(group.slug, routeParams, querystringParams) : groupDetailUrl(group.slug, routeParams, querystringParams)}
+            // TODO COMOD: groupUrl doesn't have this call signature, need to fix
```

Comment:
reminder

Reply:
Yeah, this is a bug I found, so I want to get to it in this fix. I didn't have time in that moment to verify if/how it breaks the UI